RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -1167,3 +1167,23 @@ void DAClusterizerInZ_vect::dump(const double beta, const vertex_t& y, const tra
   }
 #endif
 }
+
+void DAClusterizerInZ_vect::fillPSetDescription(edm::ParameterSetDescription& desc) {
+  desc.addUntracked<double>("zdumpcenter", 0.);
+  desc.addUntracked<double>("zdumpwidth", 20.);
+  desc.addUntracked<bool>("use_vdt", false);  // obsolete, appears in HLT configs
```

Comment:
Since you are cleaning, then it is probably now time to remove this obsolete and never used parameter
If you do so, you must allow a (temporary) customization in HLTrigger/Configuration/python/customizeHLTforCMSSW.py to remove it from the configurations in the HLT menus

Reply:
Thanks @perrotta, addressed here: https://github.com/cms-sw/cmssw/pull/33104/commits/28019810f9f0f3ff557ac5ce53603c39e5026135